pkg/project/create.go

```diff
@@ -47,6 +47,7 @@ type (
 		IgnoredPaths      []string `json:"ignoredPaths"`
 		MavenProfiles     []string `json:"mavenProfiles,omitempty"`
 		MavenProperties   []string `json:"mavenProperties,omitempty"`
+		PingTimeout       string   `json:"pingTimeout"`
```

Comment:
Should we name it `StatusPingTimeout` for more clarity?

Reply:
updated